packages/phasor/src/elements/connector/connector-element.ts

```diff
@@ -120,6 +120,10 @@ export class ConnectorElement extends SurfaceElement<IConnector> {
       this.strokeStyle === StrokeStyle.Dashed
     );
 
+    // points might not be build yet in some senarios
+    // eg. undo/redo, copy/paste
+    if (!points.length) return;
```

Comment:
I think we should make sure connector points must exists, if not , there must something wrong.